flake.nix

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
I find this a bit invasive, like for a while years ago everyone was committing ide configs into repos until it became best practice ***not*** to do that.